Kudu.Contracts/Deployment/DeploymentInfoBase.cs

```diff
@@ -61,5 +61,12 @@ public bool IsValid()
         }
 
         public abstract IRepository GetRepository();
+
+        // If this is not set, sync triggers will look under d:\home\site\wwwroot
+        // for functionsRoot. Otherwise it'll use this path for that
+        // This is used in Run-From-Zip deployments where the contnent of wwwroot
```

Comment:
Typo: 'content'

Reply:
reminded me I need to install a spell checker :)